Justfile

```diff
@@ -16,7 +16,7 @@ workbox_config := "workbox.config.cjs"
 # -----
 
 @default: dev-build
-	just dev-server & just watch
+	just dev-server # & just watch
```

Comment:
Was this change intentional?

Reply:
Ugh kind of, disabled watching because of a Nix issue `watchexec` is no longer able to compile on ARM Macs (something something Apple SDK also causing issues with `cargo watch`)

Reply:
I'll revert the change.